tests/qgis-projects/tests/replace_database_connexion_qgs_project.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+DIR="$1"
```

Comment:
* Add a docstring explaining how to call the script, the order of the parameters to give
* Related to the previous comment, let's give everything as parameters : host, login, password, dbname
* Bonus, you can count the number of parameters and if not correct, print the help

Reply:
Yes I write this